common/lib/api-client/models/populate.js

```diff
@@ -0,0 +1,9 @@
+const population = require('./population')
+module.exports = {
```

Comment:
If this also works then it's fine, but I think this method actually feels more confusing when looking at it. 

Can we guarantee that we always want to include all the fields from the population model? I just wonder if changing that model in the future would have any side-effects on this now that we're extending it.